packages/app/components/Queue/TA/TAQueueList.tsx

```diff
@@ -181,6 +181,7 @@ export default function TAQueueList({
                 {isStaffCheckedIn ? (
                   <CheckOutButton
                     danger
+                    disabled={isHelping}
```

Comment:
Should we do a tooltip to explain? Normally we tooltip disabled buttons.
eg
https://github.com/sandboxnu/office-hours/blob/d3e7dd18abcf60b98863cd15c384846bd09a5d68/packages/app/components/Queue/TA/TAQueueList.tsx#L162-L166

Reply:
rip lol

we were running into issues because of the styled component and decided it would be better in the interest of time to leave out the tooltip, not sure if there's a good way to implement it

Reply:
Yeah unfortunately the tooltip was not playing nice with what we had. I think because the button was a nested `QueueInfoColumnButton`. 
```
const CheckOutButton = styled(QueueInfoColumnButton)`
  color: #da3236;
`;
```

When we moved the styles from the `QueueInfoColumnButton` to the `CheckOutButton` and made it a `styled(Button)` the tooltip worked but that seemed too hacky and would make the code worse.

Reply:
Seems right. We'll have to revisit this eventually